DKozii/HW6/task61.py

```diff
@@ -0,0 +1,17 @@
+even_num_devisible_2 = evennum = []
```

Comment:
look at this code
even = []
odd = []
other = []

for number in range(1, 11):
    if number >= 2 and number % 2 == 0:
        even.append(number)
    elif number >= 3 and number % 3 == 0:
        odd.append(number)
    else:
        other.append(number)

output need to be
[2, 4, 6, 8, 10] is even multiple of 2
[3, 9] is an odd multiple of 3
[1, 5, 7] not divisible by 2 and 3